lc-hax/Scripts/Modules/StunMod.cs

```diff
@@ -34,6 +34,10 @@ public sealed class StunMod : MonoBehaviour {
         if (!Setting.EnableStunOnLeftClick) return;
         if (!Helper.CurrentCamera.IsNotNull(out Camera camera)) return;
 
+        if ((Helper.LocalPlayer?.currentlyHeldObject).IsNotNull(out GrabbableObject currentItem) &&
+            (currentItem.itemProperties.isDefensiveWeapon || currentItem.TryGetComponent(out Shovel _) || currentItem.TryGetComponent(out ShotgunItem _)))
+            return;
```

Comment:
You don't want `stunclick` to work when hitting with a shovel?

Reply:
no, because it's too obvious, turrets start getting jammed in the background when you aim around, and dogs gets stuck in the stun animation when then die, it's a bit annoying

Reply:
Oh, I see.

Reply:
maybe this can be revisited later if the anim game bugs are fixed, or maybe have the stun on a different button like `F` / `M4` / `M5`, idk